crml/staking/src/lib.rs

```diff
@@ -1191,7 +1191,9 @@ decl_module! {
 		#[weight = SimpleDispatchInfo::FixedNormal(5_000)]
 		fn set_invulnerables(origin, validators: Vec<T::AccountId>) {
 			ensure_root(origin)?;
-			<Invulnerables<T>>::put(validators);
+			<Invulnerables<T>>::put(validators.clone());
+			validators.iter().for_each( |v| debug::print!("Set invulnerable:{:?}", v) );
```

Comment:
[Option] Maybe something like this is also okay (just to avoid printing many lines): `debug::print!("Set invulnerables: {:?}", validators);`?

Reply:
yeah true

Reply:
I'm assuming that each invulnerable is a 32-byte address? the print may look messy no matter how we print it. It would be awesome if we could do something like "Set invulnerables: 4ab8896...4c, 6ccb23...67", but that's definately out of scope =D

Reply:
changed this to a single call